### PR TITLE
Concepts docs and new FCL Quickstart

### PR DESCRIPTION
**Pull request by 10thfloor**
Lots of changes and improvements, including a nice makeover of the homepage.

### CODE REVIEW

File: docs/content/concepts/index.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Flow Concepts
+---
+
+There's a lot to cover when we're talking about a system like Flow, the technology behind the network
+and the design of systems you'll use to interact with it. This section will help you uncover informationalthat can help you better understand
```
Comment (joshuahannan) [inline docs/content/concepts/index.mdx:6]:
```suggestion
and the design of systems you'll use to interact with it. This section will help you uncover information that can help you better understand
```

File: docs/content/flow-js-sdk/flow-app-quickstart.mdx [hunk 2]
```diff
@@ -1,629 +1,1271 @@
...
-```shell
-npx create-react-app my-dapp
+```sh
+yarn create react-app my-app
+cd my-app
+yarn add @onflow/fcl @onflow/types
 ```
 
-## Install `@onflow` Dependencies
+Those deps:
```
Comment (joshuahannan) [inline docs/content/flow-js-sdk/flow-app-quickstart.mdx:39]:
```suggestion
Those dependencies:
```

**Review by joshuahannan — COMMENTED** [review]
Looks good! I noticed that the Cadence introduction page was removed in a recent PR (not sure which one). Can you add it back? 
https://flow-docs-git-josh-qcflip.onflow.vercel.app/cadence/tutorial/00-introduction/
I found it in an unrelated PR there.

**Comment by joshuahannan** [comment]
I also wasn't able to review the dapp quickstart because I am not familiar with JS

**Comment by 10thfloor** [comment]
> Looks good! I noticed that the Cadence introduction page was removed in a recent PR (not sure which one). Can you add it back?
> https://flow-docs-git-josh-qcflip.onflow.vercel.app/cadence/tutorial/00-introduction/
> I found it in an unrelated PR there.

Hey, it wasn;t removed, just renamed. It's now the 'index' page when you land on the Cadence section.

**Comment by joshuahannan** [comment]
@10thfloor Can you put it in the sidebar also? It should be easily accessible from anywhere in the cadence section of the docs

File: docs/content/flow-port/index.md [hunk 3]
```diff
@@ -1,34 +1,36 @@
 ---
-title: Flow Port FAQ
-description: How to use the Flow Port to manage your Flow account
+title: Flow Port
+description: How to use the Flow Port
```
Comment (psiemens) [inline docs/content/flow-port/index.md:3]:
```suggestion
description: How to use Flow Port
```

File: docs/content/flow-port/index.md [hunk 4]
```diff
@@ -1,34 +1,36 @@
...
 ---
+
+Welcome to Flow Port, your portal to the decentralized world of Flow. Access your Flow account, interact with the blockchain, manage your assets and more.
 ## Creating an Account
-In order to access Flow Port, you must have a valid Flow address. If you do not have a Flow address, you can choose to create one via any of our FCL Wallet Providers. 
+In order to access Flow Port, you must have a valid Flow address. If you do not have a Flow address, you can choose to create one via any of our FCL Wallet Providers.
```
Comment (psiemens) [inline docs/content/flow-port/index.md:8]:
```suggestion
In order to access Flow Port, you must have a valid Flow address. If you do not have a Flow address, you can choose to create one via any of the available FCL wallet providers.
```

File: docs/content/flow-port/index.md [hunk 5]
```diff
@@ -50,18 +52,20 @@ In order to access Flow Port, you must have a valid Flow address. If you do not
...
- 
+
  5. Click on Create an account. Follow the prompts on your Ledger device.
- 
+
  6. Once your account address is created, you will be automatically logged into Flow Port.
 
 ## Staking & Delegating
+
+For a detailed walkthrough on how to use the Flow Port for staking and delegating, please read the [Flow Port staking walkthrough](/flow-port/staking-guide)
```
Comment (psiemens) [inline docs/content/flow-port/index.md:68]:
```suggestion
For a detailed walkthrough on how to use Flow Port for staking and delegating, please read the [Flow Port staking walkthrough](/flow-port/staking-guide)
```

File: docs/content/index.mdx [hunk 6]
```diff
@@ -8,28 +8,21 @@ import DocsetMenu from "./components/docset-menu";
 import { MenuTitle } from "./components/menu";
 import { theme } from "gatsby-theme-flow/src/colors";
 
-Before starting your journey on Flow we recommend reading the <a href="https://www.onflow.org/primer">Flow Primer</a> where you will get a quick introduction to how the Flow blockchain works. It will be easier to understand the concepts in these tutorials if you have an idea of how Flow works.
+Ready to start building? Take a tutorial, or have a look at an example application.
```
Comment (psiemens) [inline docs/content/index.mdx:11]:
```suggestion
Ready to start building? Take a tutorial or have a look at an example application.
```

File: docs/content/index.mdx [hunk 7]
```diff
@@ -8,28 +8,21 @@ import DocsetMenu from "./components/docset-menu";
...
 
-## Install Flow
+### 🏃‍♂️ App Quickstart Using React
 
-To work with Flow locally you'll need to install the <a href="https://github.com/onflow/flow/blob/master/docs/cli.md#flow-cli">Flow CLI</a>.
-The Flow CLI is a command-line interface that provides useful utilities for building Flow applications, including the Flow Emulator.
+Follow the [Flow App Quickstart](/flow-js-sdk/flow-app-quickstart/) for a detailed guide covering how to use the [Flow JS SDK](/flow-js-sdk) to build a serverless app that connects to Flow, using React!
 
-<MenuTitle>OSX & Linux</MenuTitle>
+### 😸 Kitty-Items Full-Stack Example App
```
Comment (psiemens) [inline docs/content/index.mdx:17]:
```suggestion
### 😸 Kitty Items Full-Stack Example App
```

File: docs/content/index.mdx [hunk 8]
```diff
@@ -8,28 +8,21 @@ import DocsetMenu from "./components/docset-menu";
...
-```shell
-sh -ci "$(curl -fsSL https://storage.googleapis.com/flow-cli/install.sh)"
-```
-
-<MenuTitle>Windows</MenuTitle>
-
-```powershell
-iex "& { $(irm 'https://storage.googleapis.com/flow-cli/install.ps1') }"
-```
+The Kitty Items application is a full-featured NFT marketplace that demonstrates what a complex full-stack application
```
Comment (psiemens) [inline docs/content/index.mdx:19]:
```suggestion
The Kitty Items application is a full-featured NFT marketplace that demonstrates what a full-stack application can look like on Flow.
```

File: docs/content/index.mdx [hunk 9]
```diff
@@ -8,28 +8,21 @@ import DocsetMenu from "./components/docset-menu";
...
+built on Flow will look like. It's designed to showcase the power of Cadence and Flow's client libraries.
+Have a look at the [Kitty Items Github repository](https://github.com/onflow/kitty-items) for more details.
```
Comment (psiemens) [inline docs/content/index.mdx:21]:
```suggestion
Have a look at the [Kitty Items GitHub repository](https://github.com/onflow/kitty-items) for more details.
```

**Review by psiemens — COMMENTED** [review]
Nice! The homepage is already looking much better

**Pull request merged**